server/game/cards/01-DTR/JonLongstride.js

```diff
@@ -0,0 +1,24 @@
+const DudeCard = require('../../dudecard.js');
+
+class JonLongstride extends DudeCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Jon Longstride',
+            playType: 'noon',
+            target: {
+                activePromptTitle: 'Choose horse',
+                cardCondition: { location: 'play area', condition: card => card.parent === this && card.hasKeyword('Horse') && card.booted },
+                cardType: ['goods']
+            },
+            message: context =>
+                this.game.addMessage('{0} uses {1}\'s ability to unboot {1}\'s Horse', context.player, this),
+            handler: (context) => {
+                this.controller.unbootCard(context.target);
```

Comment:
```suggestion
                this.controller.unbootCard(context.target);
                context.target.resetAbilities();
```

Also need to reset ability usage so it can be used again.